cmd/lakefs/cmd/run.go

```diff
@@ -91,6 +93,13 @@ var runCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		logger := logging.Default()
 		cfg := loadConfig()
+		viper.WatchConfig()
+		viper.OnConfigChange(func(in fsnotify.Event) {
+			lvl := viper.GetString(config.LoggingLevelKey)
```

Comment:
Can we (additionally) read the rest of the config, and warn if something else changed?

Reply:
Opening an issue, there's a lot to do here (save the previous config, compare, take defaults into account while only the logging level can really change), while the value is limited IMO (Better visibility for items that don't have any effect).